lighthouse-extension/app/src/popup.js

```diff
@@ -40,25 +58,49 @@ document.addEventListener('DOMContentLoaded', _ => {
 
   const stopSpinner = _ => {
     spinnerAnimation.cancel();
-    statusEl.classList.remove('status--visible');
+    statusEl.classList.remove(subpageVisibleClass);
   };
 
   const logstatus = ([, message, details]) => {
     statusMessageEl.textContent = message;
     statusDetailsMessageEl.textContent = details;
   };
 
+  const getAuditsOfName = name => {
+    let aggregation = aggregations.filter(aggregation => aggregation.name === name);
+
+    if (!aggregation.length) {
+      return [];
+    }
+
+    // This checks if we have a a criteria property, it's necessary to check categories like Best Practices
+    if (!aggregation[0].hasOwnProperty('criteria')) {
+      aggregation = aggregation[0].items;
+    }
+
+    return Object.keys(aggregation[0].criteria);
+  };
+
   background.listenForStatus(logstatus);
 
   generateReportEl.addEventListener('click', () => {
     startSpinner();
     feedbackEl.textContent = '';
-    background.runAudits({
-      flags: {
-        mobile: true,
-        loadPage: true
-      }
-    })
+
+    const audits = _flatten(
+      Array.from(optionsEl.querySelectorAll(':checked'))
+        .map(input => getAuditsOfName(input.value))
+    );
+
+    background.runAudits(
+      {
```

Comment:
you can nuke the line breaks around these outer brackets.  ... =>  `background.runAudits({ `